picard/util/webbrowser2.py

```diff
@@ -23,65 +23,6 @@
 from PyQt5 import QtWidgets
 from picard.const import PICARD_URLS
 
-"""
-A webbrowser extension that respects user's preferred browser on each
-platform. Python 2.5 already has *some* support for this, but it's not
-enough, in my opinion. See also:
-http://sourceforge.net/tracker/index.php?func=detail&aid=1681228&group_id=5470&atid=105470
-"""
```

Comment:
Please replace docstring instead of removing it. It's hard enough to get documentation into the codebase, so let's not remove what we have without at least putting new documentation in place.